### DISCUSSION TITLE
How to override default eventEmitInterval (10 minutes)?

### DISCUSSION THREAD

**Discussion by y2241** [redis/lettuce#2301] [General]
I am using Lettuce client 6.2.0. I tried to change the eventEmitInterval, but it does not seem to work. What have I done wrong in below code?

ClientResources buildClientResources() {
      
        ClientResources.Builder builder = DefaultClientResources.builder();
        if (isMetricsEnabled()) {
            builder.commandLatencyRecorder(new DefaultCommandLatencyCollector(DefaultCommandLatencyCollectorOptions.builder()
                            .resetLatenciesAfterEvent(true)
                            .enable()
                            .targetUnit(MILLISECONDS)
                            .build()))
                    .commandLatencyPublisherOptions(DefaultEventPublisherOptions.builder().eventEmitInterval(ofSeconds(5)).build());
        }
        builder.reconnectDelay(Delay.exponential(0, 1500, MILLISECONDS, 2));
        return builder.build();
    }

RedisClient client = RedisClient.create(buildClientResources(), uri);

**Comment by y2241** [comment]
It always sends metrics once every 10 minutes, instead of once every 5 seconds.

**Comment by mp911de** [comment]
I'm not able to reproduce the failure. The given code above configures `ClientResources` that emits metric events every 5000ms.

**Reply by y2241** [reply]
yes, you are right. It does generate metrics every 5 seconds (my settings) on our servers. I just cannot do it on my localhost for some reason.
Thanks for your response.
